src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Fruit> fruits = new HashMap<>();
```

Comment:
According to the encapsulation principle, the visibility of `fruits` should be restricted. Instead of making it public, consider making it private and providing public methods to interact with it, such as getters and setters or specific methods to add and retrieve fruits.

Reply:
done